tests/cloudferrylib/os/identity/test_keystone.py

```diff
@@ -120,6 +120,22 @@ def test_get_client_generates_new_token(self):
         client2 = self.keystone_client.keystone_client
         self.assertFalse(client1 == client2)
 
+    @mock.patch("time.sleep")
+    @mock.patch.object(keystone_client, "Client")
+    def test_retry_when_get_client_with_expected_unexpected_exception(
+            self, mock_call, sleep_mock):
+        # side effect is defined with two exceptions during get_client :
+        # One is Expected - exceptions.Unauthorized
+        # Another is Unexpected -  exceptions.AuthorizationFailure
```

Comment:
Please use doc-string to describe or document testing methods. In that case we can use them for documentation generation.